### ISSUE TITLE
Prioritize score not readable with screenreader

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode-pull-request-github#6371]
Testing #6329

On Windows, I have microsoft/vscode open and prioritized the notifications.
I then tried navigating the Notifications section with NVDA's speech visualizer open. I found that I could reveal and read the hover with Ctrl+K Ctrl+I, but that I was not able to see the notification score show up in the speech visualizer while navigating around.

**Comment by alexr00** [comment]
I think that score is intended for debugging purposes. 

However, it just uses decorations for the score. I think we should just make sure the score is in the hover.
